imperial_coldfront_plugin/gpfs_client.py

```diff
@@ -202,3 +204,20 @@ def set_quota(
             filesGracePeriod="null",
             blockGracePeriod="null",
         )
+
+
+def background_task(func):
+    """Wrapper to run a function in the background."""
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        """Run the function in the background."""
+        return async_task(func, *args, **kwargs)
+
+    return wrapper
```

Comment:
Rather than copying this from `ldap.py` maybe lets make it a utility function in `tasks.py`.